opencog/nlp/chatbot/sentence-matching.scm

```diff
@@ -0,0 +1,32 @@
+; Returns one or more sentences that are structurally similar to the input one.
```

Comment:
Yeah, I didn't review and am not critiquing the algorithm. I only reviewed superficial details, e.g. does it fit? is it well organized? Cleanly coded? etc. Mostly its clean, it fits and is well organized. If the algorithm is horrible, we'll find out later.